packages/block-editor/src/components/block-tools/zoom-out-mode-inserters.js

```diff
@@ -51,8 +51,7 @@ function ZoomOutModeInserters() {
 		}
 		// reset insertion point when the block order changes
 		setInserterIsOpened( true );
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [ blockOrder ] );
+	}, [ blockOrder, setInserterIsOpened ] );
```

Comment:
I'm pretty sure I added this because it was misbehaving, but I can't remember after all the changes. What happens theoretically when the function changes? We only want to reset it when the order changes, so maybe we have to check that in the effect.

Reply:
Will this ever happen? It looks like a stable reference to me, so I don't understand when it will change. Plus we also have the mounted check. Happy to abandon it if I'm missing something of course.

Reply:
Right, actions should be safe to list in dependencies. If they're not anymore, we might have a bigger problem 😅

P.S. It would be cool if linter treated Redux actions as state setters, but IIRC that involved a lot of work.

Reply:
Sure, if everything works fine. I could swear I added it for a reason 😅

Reply:
We don't know it's an action though, it's a setting, right? I don't love these "impure" dependencies that much tbh

Reply:
Yes. It's a setting pointing to the Editor package's `setIsInserterOpened` [action](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/provider/use-block-editor-settings.js#L314).

For reference, this was the commit that disabled ESLint - ac1bbadb47308ffd0e1ffea7f0a7a91f254b9a0e.

Reply:
What George said. Thanks for verifying it, folks 🙌